test/generate-testcase.js

```diff
@@ -54,7 +55,10 @@ function fetchSource(url, callbackFn) {
   if (url.indexOf("https") == 0) {
     client = require("https");
   }
-  client.get(url, function(response) {
+  var options = urlparse(url);
+  options.headers = {'User-Agent': 'readability.js'}
```

Comment:
This seems fine to me, but I would prefer to send something more resembling Firefox. I know that it's lying, but a surprising number of sites adjust content that you get, and we should probably try to get content that vaguely resembles what Firefox gets.